flogger/README.md

```diff
@@ -1,5 +1,39 @@
-# Google Flogger
+## Google Flogger
 
-This module contains Flogger [sources][flogger-github] built with Gradle.
+This directory contains Flogger [modules][flogger-github] built with Gradle.
```

Comment:
We need to speak about repackaging we performed somewhere. This file seems to be a proper place.